.github/workflows/npmPublishSdk.yml

```diff
@@ -15,6 +15,8 @@ jobs:
       - run: yarn test:ci
 
       - name: Publish to NPM
-        run: cd @stellar/typescript-wallet-sdk && yarn publish --access public
```

Comment:
not changing anything, but it's better w/ pipe than `&&`

Reply:
Out of curiosity -- why is it better?